mmv1/third_party/terraform/provider/provider.go.erb

```diff
@@ -113,6 +115,11 @@ func Provider() *schema.Provider {
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
 
+			"universe_domain": {
```

Comment:
Please write user-facing documentation for the new property. A basic description is fine- it should go under https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/provider_reference#advanced-settings-configuration

Reply:
sure will do